web-wallet/src/lib/dusk/components/Mnemonic/Mnemonic.svelte

```diff
@@ -118,63 +121,67 @@
 </script>
 
 <div {...$$restProps} class={classes}>
-  <div class="dusk-mnemonic__actions-wrapper">
-    {#if type === "authenticate" && shouldShowPaste}
+  {#if !isTriggeredByLogin}
+    <div class="dusk-mnemonic__actions-wrapper">
+      {#if type === "authenticate" && shouldShowPaste}
+        <Button
+          icon={{ path: mdiContentPaste }}
+          text="Paste seed phrase"
+          variant="tertiary"
+          on:click={pasteSeed}
+        />
+      {/if}
       <Button
-        icon={{ path: mdiContentPaste }}
-        text="Paste seed phrase"
+        disabled={!currentIndex}
+        on:click={undoLastWord}
+        icon={{ path: mdiRedoVariant }}
+        text="Undo"
         variant="tertiary"
-        on:click={pasteSeed}
       />
-    {/if}
-    <Button
-      disabled={!currentIndex}
-      on:click={undoLastWord}
-      icon={{ path: mdiRedoVariant }}
-      text="Undo"
-      variant="tertiary"
-    />
-  </div>
+    </div>
+  {/if}
 
   <Words words={enteredMnemonicPhrase} />
```

Comment:
As this is the only common bit, maybe it's easier to just have an "if / else" instead of two "ifs" in different spots?
e.g.

```svelte
{#if isTriggeredByLogin}
  <Words words={enteredMnemonicPhrase} />
{:else}
  <!-- ... -->
{/if}
```

Reply:
I'm not sure, because one of the bits appear above and the another – below the words. If we follow you approach, it  means that then the `<Words words={enteredMnemonicPhrase} />` will have to be repeated twice (one in the `if` and then once in the `else` statement).

Reply:
> I'm not sure, because one of the bits appear above and the another – below the words. If we follow you approach, it means that then the `<Words words={enteredMnemonicPhrase} />` will have to be repeated twice (one in the `if` and then once in the `else` statement).

Yes, but it seems easier to read at my eyes.

Do as you prefer, anyway, as long as tests catch both situations.